jrodriguezdiazz/jrodriguezdiazz-portfolio
# Add email send functionality

## Description
- Add email send functionality
  - Add dependency: `emailjs-com` and `sweetalert2`
  - Add environment variables for captcha and email service
  - Rename the function `pageview` to `pageView`
  - Add a file that handles environment variables (`lib/getEnv.js`)
  - Change styles of contact form inputs
  - Add validations to form inputs



---

## Motivation and Context

The purpose of these changes is to add the functionality of sending email to have a contact with the users of the page





---

## How Has This Been Tested?
- Go to: https://jrodriguezdiazz-portfolio-261s176bs-jrodriguezdiazz.vercel.app/contact
- Fill all the input
  - Take into consideration the following comments:
    - All the input are required
    - The subject, username and email are validate with regular expression
- You should see an alert that says the email has been sent






---

## Screenshots (if appropriate):

| Desktop      | Mobile |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/24766287/141660464-945d04e1-679f-4842-8e68-4dd7c0c21e25.png) | ![image](https://user-images.githubusercontent.com/24766287/141660420-7fe624a2-a41d-45e3-80e1-027ab3be33f1.png) |



---

## Types of Changes




---

## Checklist:





---

# Other Notes
